!Command
command:
- geneChecker_fasta_gaps.py
positional: []
named: []
parent:
subcommands: []
usage: []
help_flag:
usage_flag:
version_flag:
help_text: "Traceback (most recent call last):\n  File \"/usr/local/bin/geneChecker_fasta_gaps.py\"\
  , line 509, in <module>\n    blasteVal=sys.argv[7]\nIndexError: list index out of\
  \ range\n"
generated_using:
- --help
docker_image: quay.io/biocontainers/mitofinder:1.4--py27h516909a_0
